src/ScheduleService.Models.Db/DbGroup.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.ScheduleService.Models.Db;
+
+public class DbGroup
+{
+  public const string TableName = "Groups";
+
+  public Guid Id { get; set; }
+  public string Name { get; set; }
+  public Guid WorkspaceId { get; set; }
```

Comment:
same

Reply:
you mean like this?

  public Guid Id { get; set; }
  public Guid WorkspaceId { get; set; }
  public string Name { get; set; }

Reply:
+

Reply:
+